website/docs/release_notes.md

```diff
@@ -8,6 +8,187 @@ import ReactPlayer from 'react-player'
+# [INFO] Total time:  2.589 s
+# [INFO] Finished at: 2024-08-22T12:08:36+02:00
+# [INFO] ------------------------------------------------------------------------
+```
+
+Added by [@yadavan88](https://github.com/yadavan88) in [#3003](https://github.com/VirtusLab/scala-cli/pull/3003).
+
+### Support for launching apps from dependencies without other inputs
+It is now possible to launch an app by just specifying its dependency, without the need to provide any source files.
+In such a case the build server will not be started, as there's no compilation to be done.
+There's also no need to specify the main class, as it's now being detected automatically in dependencies as well.
+Do note that explicitly calling the `run` sub-command is necessary here, as otherwise Scala CLI will default to the REPL.
+
+```bash
+scala-cli run --dep io.get-coursier:coursier-cli_2.13:2.1.10 -- version
+# 2.1.10
+```
+
+This can be used similarly to [Coursier's `cs launch`](https://get-coursier.io/docs/cli-launch).
+
+Added by [@kasiaMarek](https://github.com/kasiaMarek) in [#3079](https://github.com/VirtusLab/scala-cli/pull/3079).
+
+### (⚡️ experimental) JMH available in various commands and via `using` directives
+Some improvements have been done to the experimental support for JMH (Java Microbenchmark Harness).
+
+The `--jmh` and `--jmh-version` options can now be passed to a number of commands:
+- `compile`, so that a Scala CLI project with benchmarking can be compiled separately from being run.
+- `package`, although the resulting artifacts will run the project as normal for now, rather than benchmarks.
+- `setup-ide`, so that benchmarking projects can be imported to your IDE of choice.
+- `test` and `export` will now also no longer fail with `--jmh`, although no specific implementations for benchmarking are in place there yet.
+
+It is now also possible to control JMH with `using` directives:
+- `//> using jmh` allows to enable JMH for the project, being the equivalent of the `--jmh` option.
+- `//> using jmhVersion <version>` allows to set the JMH version to use, being the equivalent of the `--jmh-version` option.
+
+```scala compile power
+//> using jmh
+//> using jmhVersion 1.37
+package bench
+
+import org.openjdk.jmh.annotations.*
+import java.util.concurrent.TimeUnit
+
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@Warmup(iterations = 1, time = 100, timeUnit = TimeUnit.MILLISECONDS)
+@Measurement(iterations = 10, time = 100, timeUnit = TimeUnit.MILLISECONDS)
+@Fork(0)
+class Benchmarks {
+  @Benchmark
+  def foo(): Unit = {
+    (1L to 1000L).sum
+  }
+}
+```
+
+Expect more improvements in this area in the future.
+Also, do play with it and give us feedback about the current implementation!
+
+Added by [@Gedochao](https://github.com/Gedochao) in [#3091](https://github.com/VirtusLab/scala-cli/pull/3091) and [#3118]( and https://github.com/VirtusLab/scala-cli/pull/3118).
```

Comment:
```suggestion
Added by [@Gedochao](https://github.com/Gedochao) in [#3091](https://github.com/VirtusLab/scala-cli/pull/3091) and [#3118](https://github.com/VirtusLab/scala-cli/pull/3118).
```

Reply:
The regex caught this wrong... Making a mental note to fix the automation later.